### PR TITLE
Remediations card

### PR DESCRIPTION
**Pull request by ryelo**
## Prerequisites

- [x] Scope your styles to your individual card
- [x] You are using translations when necessary with proper plural/singular modifications
- [x] You are using the template card provided
- [x] Use functional components & hooks (please)
- [x] Attach screenshots (before and after)
- [x] Make sure that all text in blue is populated with the correct link. If you're unsure what the url should be, just ask!

## What

Adds the remediations card and empty states

### Normal
Screen Shot 2020-04-13 at 3 30 23 PM

Screen Shot 2020-04-13 at 3 52 53 PM

### No data
Screen Shot 2020-04-13 at 3 32 34 PM

Screen Shot 2020-04-13 at 3 51 23 PM

^ This is the same state that we use in remediations

## Additional Info
We are not doing translations for the playbook names

Also bumped the min-width of the status to accommodate for the "No activity"

## Code Review
@christiemolloy @ryelo @AllenBW

## Design Review
@kybaker

### CODE REVIEW

File: src/SmartComponents/Remediations/RunStatus.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    useEffect(() => {
+        const fetchPlaybookRun = async () => {
+            try {
+                const response = await API.get(`${REMEDIATIONS_PLAYBOOK_RUNS_FETCH_URL}${id}/playbook_runs?limit=1&offset=0&sort=-updated_at`);
+                setPlaybookRun(response.data.data);
+                setLoaded(true);
+                console.error(response.data);
+            } catch (error) {
+                setLoaded(false);
+                console.error('oopsie');
```
Comment (AllenBW) [inline src/SmartComponents/Remediations/RunStatus.js:27]: but yah might want to not do this

File: package.json [hunk 2]
```diff
@@ -51,6 +51,7 @@
     "@patternfly/react-table": "2.28.28",
     "@patternfly/react-tokens": "2.8.11",
     "@red-hat-insights/insights-frontend-components": "3.41.70",
+    "@redhat-cloud-services/frontend-components": "^1.0.14",
```
Comment (AllenBW) [inline package.json:54]: if possible, lets keep this locked down to exact versions

File: src/AppActions.js [hunk 3]
```diff
@@ -67,3 +67,13 @@ export const subscriptionsUtilizedProductTwoFetch = (id, options) => ({
         fetchData(`${ActionTypes.SUBSCRIPTIONS_UTILIZED_CAPACITY_FETCH_URL}${id}`, {}, options)
     ])
 });
+
+export const fetchRemediations = (options) => ({
+    type: ActionTypes.REMEDIATIONS_FETCH,
+    payload: fetchData(ActionTypes.REMEDIATIONS_FETCH_URL, {}, options)
+});
+
+export const fetchRemediationsPlaybookRuns = (id, options) => ({
```
Comment (AllenBW) [inline src/AppActions.js:76]: don't need this one any more <3

File: src/AppReducer.js [hunk 4]
```diff
@@ -25,7 +25,11 @@ const initialState = Immutable({
     subscriptionsUtilizedProductOne: [],
     subscriptionsUtilizedProductOneFetchStatus: '',
     subscriptionsUtilizedProductTwo: [],
-    subscriptionsUtilizedProductTwoFetchStatus: ''
+    subscriptionsUtilizedProductTwoFetchStatus: '',
+    remediations: {},
+    remediationsFetchStatus: '',
+    remediationsPlaybookRuns: {},
```
Comment (AllenBW) [inline src/AppReducer.js:31]: don't need this one or the following one

File: src/AppReducer.js [hunk 5]
```diff
@@ -155,5 +159,26 @@ export const DashboardStore = (state = initialState, action) => {
...
+            return state.set('remediationsFetchStatus', 'pending');
+        case `${ActionTypes.REMEDIATIONS_FETCH}_FULFILLED`:
+            return Immutable.merge(state, {
+                remediations: action.payload,
+                remediationsFetchStatus: 'fulfilled'
+            });
+        case `${ActionTypes.REMEDIATIONS_FETCH}_REJECTED`:
+            return state.set('remediationsFetchStatus', 'rejected');
+
+        case `${ActionTypes.REMEDIATIONS_PLAYBOOK_RUNS_FETCH}_PENDING`:
```
Comment (AllenBW) [inline src/AppReducer.js:174]: can kill this whole block

**Comment by kybaker** [comment]
@ryelo Do we use an icon on the other 'not created' screens? If not can you just remove it? Other than that looks great!

**Comment by christiemolloy** [comment]
Can you double check that the empty state is taking up full-width of its parent container? It seems to have a width on it that is limiting it from stretching the full way

Screen Shot 2020-04-14 at 1 40 00 PM

**Comment by ryelo** [comment]
@kybaker I'm not sure about that one. It's the current design we use in remediations. I can remove it if you'd like though.

**Comment by ryelo** [comment]
@christiemolloy I actually fixed that in https://github.com/RedHatInsights/insights-dashboard/pull/112/commits/98b12f3f98110569f685fa4dcfa8dc0cc3576fe1, but didn't update the screenshot

**Comment by christiemolloy** [comment]
ok great! @ryelo

**Comment by kybaker** [comment]
> @kybaker I'm not sure about that one. It's the current design we use in remediations. I can remove it if you'd like though.

Lets go with no icon.

**Comment by ryelo** [comment]
Updated screenshot:

Screen Shot 2020-04-14 at 2 42 26 PM

**Pull request merged**